uint8 id                  # 1 ~ 8
uint16 angle              # 0 ~ 8191
int16 rotational_speed   # rpm
int16 current
uint8 motor_temperature  # â„ƒ

